### PR TITLE
Minecraft server domain

### PR DESCRIPTION
**Pull request by 3geETR**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

Domain for the server i play with my friends.

### CODE REVIEW

**Review by omsenjalia — DISMISSED** [review]
Bunch of grey spaces in here and commenting for the first file: that's now how mc records work.

**Comment by 3geETR** [comment]
> Bunch of grey spaces in here and commenting for the first file: that's now how mc records work.

I updated the file, should I do it like this?

File: domains/mc.3gee.json
Comment (wdhdev) [inline domains/mc.3gee.json:1]: You don't need this file if you're using Minecraft.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/mc.3gee.json
Comment (3geETR) [reply domains/mc.3gee.json:1]: If the port is different from 25565, it must be done through the srv record.

**Review by 3geETR — COMMENTED** [review]
If the port is different from 25565, it must be done through the srv record.

File: domains/mc.3gee.json
Comment (3geETR) [reply domains/mc.3gee.json:1]: Ok

**Review by wdhdev — APPROVED** [review]

**Comment by wdhdev** [comment]
> If the port is different from 25565, it must be done through the srv record.

That is correct.

**Pull request merged**